### PR TITLE
Update usage message for pscoast

### PR DESCRIPTION
**Pull request by PaulWessel**
As per #5341.  I had to add an argument to _gmt_pen_syntax_ which explains all the files with calls getting an extra NULL argument.  This was so I could use that function to prepend `<feature>/`.  This PR also wraps the usage of _gmt_DCW_option_.

### CODE REVIEW

**Comment by meghanrjones** [comment]
@PaulWessel, do you mind resolved the merge conflicts?

**Comment by PaulWessel** [comment]
Let's see if I can merge in master on that one on this end.

**Comment by PaulWessel** [comment]
OK, that worked fine - had to do with a function that gained another argument.

**Comment by PaulWessel** [comment]
You should be able to build and review it now.

File: src/pscoast.c [hunk 1]
```diff
@@ -176,94 +176,88 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct PSCOAST_CTRL *C) {	/* Deallo
...
-	GMT_Message (API, GMT_TIME_NONE, "\t[-M] [-N<feature>[/<pen>]] %s%s[-Q] [-S[<fill>]]\n", API->O_OPT, API->P_OPT);
-	GMT_Message (API, GMT_TIME_NONE, "\t[-Td%s]\n\t[-Tm%s]\n", GMT_TROSE_DIR, GMT_TROSE_MAG);
-	GMT_Message (API, GMT_TIME_NONE, "\t[%s] [%s] [-W[<feature>/][<pen>]]\n", GMT_U_OPT, GMT_V_OPT);
-	GMT_Message (API, GMT_TIME_NONE, "\t[%s] [%s] [%s] %s[%s]\n", GMT_X_OPT, GMT_Y_OPT, GMT_bo_OPT, API->c_OPT, GMT_do_OPT);
-	GMT_Message (API, GMT_TIME_NONE, "\t[%s] [%s] [%s] [%s]\n", GMT_p_OPT, GMT_t_OPT, GMT_colon_OPT, GMT_PAR_OPT);
-#ifdef DEBUG
-	GMT_Message (API, GMT_TIME_NONE, "\t[-+<bin>]\n");
-#endif
-	GMT_Message (API, GMT_TIME_NONE, "\n");
+	GMT_Usage (API, 0, "usage: %s %s [%s] [%s] [-C<fill>[+l|r]] [-D<resolution>][+f] [-E%s] "
```
Comment (meghanrjones) [inline src/pscoast.c:187]:
```suggestion
	GMT_Usage (API, 0, "usage: %s %s %s [%s] [%s] [-C<fill>[+l|r]] [-D<resolution>][+f]] [-E%s] "
```

File: src/pscoast.c [hunk 2]
```diff
@@ -176,94 +176,88 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct PSCOAST_CTRL *C) {	/* Deallo
...
+		"[-G[<fill>]] [-F%s] [-I<feature>[/<pen>]] %s [-L%s] [-M] [-N<feature>[/<pen>]] %s%s[-Q] [%s] [-S[<fill>]] "
```
Comment (meghanrjones) [inline src/pscoast.c:188]:
```suggestion
		"[-G[<fill>]] [-F%s] [-I<feature>[/<pen>]] %s [-L%s] [-M] [-N<feature>[/<pen>]] %s%s[-Q] [-S[<fill>]] "
```

File: src/pscoast.c [hunk 3]
```diff
@@ -176,94 +176,88 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct PSCOAST_CTRL *C) {	/* Deallo
...
+		"[-Td%s] [-Tm%s] [%s] [%s] [-W[<feature>/][<pen>]] [%s] [%s] [%s] %s[%s] [%s] [%s] [%s]%s [%s]\n",
+		name, GMT_J_OPT, GMT_A_OPT, GMT_B_OPT, DCW_OPT, GMT_PANEL, API->K_OPT, GMT_SCALE, API->O_OPT,
+		API->P_OPT, GMT_Rgeoz_OPT, GMT_TROSE_DIR, GMT_TROSE_MAG, GMT_U_OPT, GMT_V_OPT, GMT_X_OPT, GMT_Y_OPT, GMT_bo_OPT, API->c_OPT,
```
Comment (meghanrjones) [inline src/pscoast.c:191]:
```suggestion
		API->P_OPT, GMT_TROSE_DIR, GMT_TROSE_MAG, GMT_U_OPT, GMT_V_OPT, GMT_X_OPT, GMT_Y_OPT, GMT_bo_OPT, API->c_OPT,
```

File: src/pscoast.c [hunk 4]
```diff
@@ -176,94 +176,88 @@ static void Free_Ctrl (struct GMT_CTRL *GMT, struct PSCOAST_CTRL *C) {	/* Deallo
...
-	GMT_Message (API, GMT_TIME_NONE, "\t[%s] [%s] [%s] %s[%s]\n", GMT_X_OPT, GMT_Y_OPT, GMT_bo_OPT, API->c_OPT, GMT_do_OPT);
-	GMT_Message (API, GMT_TIME_NONE, "\t[%s] [%s] [%s] [%s]\n", GMT_p_OPT, GMT_t_OPT, GMT_colon_OPT, GMT_PAR_OPT);
-#ifdef DEBUG
-	GMT_Message (API, GMT_TIME_NONE, "\t[-+<bin>]\n");
-#endif
-	GMT_Message (API, GMT_TIME_NONE, "\n");
+	GMT_Usage (API, 0, "usage: %s %s [%s] [%s] [-C<fill>[+l|r]] [-D<resolution>][+f] [-E%s] "
+		"[-G[<fill>]] [-F%s] [-I<feature>[/<pen>]] %s [-L%s] [-M] [-N<feature>[/<pen>]] %s%s[-Q] [%s] [-S[<fill>]] "
+		"[-Td%s] [-Tm%s] [%s] [%s] [-W[<feature>/][<pen>]] [%s] [%s] [%s] %s[%s] [%s] [%s] [%s]%s [%s]\n",
+		name, GMT_J_OPT, GMT_A_OPT, GMT_B_OPT, DCW_OPT, GMT_PANEL, API->K_OPT, GMT_SCALE, API->O_OPT,
```
Comment (meghanrjones) [inline src/pscoast.c:190]:
```suggestion
		name, GMT_J_OPT, GMT_Rgeoz_OPT, GMT_A_OPT, GMT_B_OPT, DCW_OPT, GMT_PANEL, API->K_OPT, GMT_SCALE, API->O_OPT,
```

**Review by meghanrjones — APPROVED** [review]

**Pull request merged**